docs/articles_en/get-started/install-openvino/install-openvino-zypper.rst

```diff
@@ -0,0 +1,193 @@
+   * is dedicated to Linux users only
+   * additionally includes code samples
+
+.. tab-set::
+
+   .. tab-item:: System Requirements
+      :sync: system-requirements
+
+      | Full requirement listing is available in:
+      | :doc:`System Requirements Page <../../../about-openvino/release-notes-openvino/system-requirements>`
+
+      .. note::
+
+         OpenVINO RPM packages are compatible with and can be run on the following operating systems:
+
+         - openSUSE Tumbleweed
+
+   .. tab-item:: Processor Notes
+      :sync: processor-notes
+
+      | To see if your processor includes the integrated graphics technology and supports iGPU inference, refer to:
+      | `Product Specifications <https://ark.intel.com/>`__
+
+   .. tab-item:: Software
+      :sync: software
+
+      * `CMake 3.13 or higher, 64-bit <https://cmake.org/download/>`_
+      * GCC 8.2.0
+      * `Python 3.8 - 3.11, 64-bit <https://www.python.org/downloads/>`_
+
+
+Install OpenVINO Runtime
+########################
+
+Step 1: Set Up the Repository
++++++++++++++++++++++++++++++
+
+
+1. Create a ZYPPER repository file with the command below:
+
+   .. code-block:: sh
+
+      sudo zypper addrepo https://download.opensuse.org/repositories/science/openSUSE_Tumbleweed/science.repo
+      sudo zypper refresh
+
+2. Verify that the new repository is set up properly.
+
+   .. code-block:: sh
+
+      zypper lr |grep -i science
+
+   You will see the available list of packages.
+
+   To list available OpenVINO packages, use the following command:
+
+   .. code-block:: sh
+
+   zypper se openvino
+
+Step 2: Install OpenVINO Runtime Using the YUM Package Manager
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+
+Install OpenVINO Runtime
+-------------------------
+
+.. tab-set::
+
+   .. tab-item:: The Latest Version
+      :sync: latest-version
+
+      Run the following command:
+
+      .. code-block:: sh
+
+         sudo zypper install openvino
+
+Check for Installed Packages and Version
+-----------------------------------------
+
+
+Run the following command:
+
+.. code-block:: sh
+
+   zypper se -i openvino
+
+.. note::
+
+   You can additionally install Python API using one of the alternative methods (:doc:`conda <openvino_docs_install_guides_installing_openvino_conda>` or :doc:`pip <openvino_docs_install_guides_installing_openvino_pip>`).
+
+Congratulations! You've just Installed OpenVINO! For some use cases you may still
+need to install additional components. Check the
+:doc:`list of additional configurations <../configurations>`
+to see if your case needs any of them.
+
+With the ZYPPER distribution, you can build OpenVINO sample files, as explained in the
+:doc:`guide for OpenVINO sample applications <../../../learn-openvino/openvino-samples>`.
+For C++ and C, just run the ``build_samples.sh`` script:
+
+.. tab-set::
+
+   .. tab-item:: C++
+      :sync: cpp
+
+      .. code-block:: sh
+
+         /usr/share/openvino/samples/cpp/build_samples.sh
```

Comment:
I have installed `openvino-sample` and this command does not work after it

Reply:
Done: Intel and openSUSE convention fix

Evidence:  https://build.opensuse.org/request/show/1170090

![evidence](https://github.com/openvinotoolkit/openvino/assets/675645/a28c8dfa-75af-431f-91fe-0a60eeb0d866)

Reply:
> I have installed `openvino-sample` and this command does not work after it
![Uploading evidence.jpg…]()

Done: Intel and openSUSE convention fix

Evidence: https://build.opensuse.org/request/show/1170090